### PR TITLE
Initial Traffic Server state module

### PR DESCRIPTION
**Pull request by cedwards**

### CODE REVIEW

**Comment by jfindlay** [comment]
@cedwards, thanks for the pull request.  Would you mind also fixing the associated lint?

File: salt/states/trafficserver.py [hunk 1]
```diff
@@ -0,0 +1,393 @@
+# -*- coding: utf-8 -*-
+'''
+Control Apache Traffic Server
+=============================
+'''
```
Comment (rallytime) [inline salt/states/trafficserver.py]: Can you add a `.. versionadded:: Beryllium` tag?

**Comment by cedwards** [comment]
@jfindlay - I'm not really sure how to address the lint issues here. What is the best way to address lint issues regarding ``__salt__``, etc?

**Comment by jfindlay** [comment]
@cedwards, sorry, I tend to assume that all salt contributors are experts on salt development.  I think the problem with you `__salt__` statements is that you forgot to actually call the function, `__salt__['trafficserver.bounce_cluster']()`.

**Comment by cedwards** [comment]
I've tried as you suggested locally and I still see the same lint errors in the output.. any other ideas?

**Comment by jfindlay** [comment]
I'll try to spend some time on this tomorrow.

**Comment by jfindlay** [comment]
Go Go Jenkins!

**Comment by jfindlay** [comment]
Other than the first error, there are only two error types:
```python
    ret = {'name': name,
```
`name` is referenced but not defined, and
```python
    __salt__['trafficserver.bounce_cluster']
```
which is a statement with no effect, because the function is only being mentioned and not called.  `__salt__` is a dictionary containing all the available module functions.  In order to use one you need to call it, `__salt__['trafficserver.bounce_cluster']()`.

**Pull request merged**